### PR TITLE
Feature/53 living income benchmark n cpi master seeder

### PR DESCRIPTION
**Pull request by wayangalihpratama**
### TODO / DONE

---

- #53 
- [x] Regions data transform
- [x] Region table
- [x] Region seeder
- [x] Load all regions data to config.js 
- [x] Living income benchmark data transform
- [x] Living income benchmark alter table
- [x] Living income benchmark seeder
- [x] Living income benchmark GET all endpoint
- [x] CPI data transform
- [x] CPI data seeder

---

- #57 
- [x] Load regions 
- [x] Load benchmark and CPI data
- [x] Save target, adult, child value
- [x] Load target, adult, child initial data value

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205755696526864
  - https://app.asana.com/0/0/1205801826524028

### CODE REVIEW

File: backend/models/living_income_benchmark.py [hunk 1]
```diff
@@ -9,64 +9,88 @@
...
-        passive_deletes=True,
-        backref='country_living_income_benchmark'
-    )
-    currency_detail = relationship(
-        'Currency',
-        cascade="all, delete",
-        passive_deletes=True,
-        backref='currency_living_income_benchmark'
-    )
+    # country_detail = relationship(
```
Comment (dedenbangkit) [inline backend/models/living_income_benchmark.py:34]: Please remove this and all the comments, if it's not used

File: backend/source/master/regions.csv
Comment (dedenbangkit) [inline backend/source/master/regions.csv:1]: Why we have mixed string and integer in the array?

File: backend/routes/living_income_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+from typing import List
+
+from db.connection import get_session
+from models.living_income_benchmark import LivingIncomeBenchmarkDict
+
+security = HTTPBearer()
+lib_route = APIRouter()
+
+
+@lib_route.get(
```
Comment (dedenbangkit) [inline backend/routes/living_income_benchmark.py:19]: the API should be filter by region id

File: backend/routes/cpi.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+def get_cpi_by_country_year(
+    req: Request,
+    country: int,
+    year: Optional[int] = None,
+    session: Session = Depends(get_session),
+):
+    res = crud_cpi.get_cpi_by_country_year(
+        session=session, country=country, year=year
+    )
+    return [r.serialize for r in res]
```
Comment (dedenbangkit) [inline backend/routes/cpi.py:35]: It should return an object. It seems you didn't implement unique between country and year

**Comment by wayangalihpratama** [comment]
Hi mas @dedenbangkit , 
I have tried to match the CPI country with our countries.csv, 
But I can't find the below countries name on our countries.csv
- Curaçao, Kingdom of the Netherlands
- Kosovo, Rep. of
- South Sudan, Rep. of
- West Bank and Gaza

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**